x-pack/plugins/security_solution/public/management/components/endpoint_responder/get_processes_action.tsx

```diff
@@ -201,7 +201,11 @@ export const GetProcessesActionResult = memo<
       >
         <FormattedMessage
           id="xpack.securitySolution.endpointResponseActions.getProcesses.errorMessage"
-          defaultMessage="The following errors were encountered: {errors}"
+          defaultMessage="The following errors were encountered:"
+        />
+        <FormattedMessage
```

Comment:
Why do we need a `<FormattedMessage />` for this if there is no custom message in, only the errors?

Reply:
You're correct, I removed the `FormattedMessage` .  I also realized that the error block was repeated in all of the components, so I split it out and updated each action with this commit: b5d7ca467565f1c6bcd65211b2f137a3593d46e1

Let me know if you have questions.